app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
+  has_attached_file :image, styles: { medium: '300x300>', thumb: '100x100>' }, default_url: 'user_image.png'
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.